### PR TITLE
Cosmetic modifications in profile delta sample string

### PR DESCRIPTION
**Pull request by sfuchs79**
### Describe the pull request:

- [x] Bug fix
- [ ] Functional change
- [ ] New feature
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Pull request long description:

Consistently do not use a space between value and unit.
Consistently do not use a space between "name:" and value.
Add "/min" for SAC rate.

### CODE REVIEW

**Comment by sfuchs79** [comment]
@dirkhh 
After merging this one (if it is accepted) it would be great to generate Transifex source strings because with the latest items all counted together we changed a couple of strings.

**Review by neolit123 — APPROVED** [review]

File: core/profile.c [hunk 1]
```diff
@@ -1660,7 +1660,7 @@ void compare_samples(struct plot_data *e1, struct plot_data *e2, char *buf, int
 			int sac = lrint(volume_used / atm * 60 / delta_time);
 			memcpy(buf2, buf, bufsize);
 			volume_value = get_volume_units(sac, &volume_precision, &volume_unit);
-			snprintf_loc(buf, bufsize, translate("gettextFromC", "%s SAC: %.*f%s"), buf2, volume_precision, volume_value, volume_unit);
+			snprintf_loc(buf, bufsize, translate("gettextFromC", "%s SAC:%.*f%s/min"), buf2, volume_precision, volume_value, volume_unit);
 		}
```
Comment (neolit123) [inline core/profile.c:1664]: could you please explain your motivation for removing the space after the `:` in these two instances.
the rest of the file seems to use space after the `:` character more consistently.

**Review by neolit123 — CHANGES_REQUESTED** [review]
added question about the spaces after `:`.

File: core/profile.c (hunk 1 above)
Comment (sfuchs79) [reply core/profile.c:1664]: @neolit123 
Yes, you are right: In the profile info box we have "name: xx.xvalue" with space consistently.
In this "delta sample string" we had a mix up to now with space (2 occurrences) and w/o space (8 occurrences). I guess the reason why w/o space may be preferred here is to limit the length of the string as much as possible.
I think that this small mismatch between layout in profile info box and this delta sample string is ok. I'll show you one picture:
grafik

Comment (neolit123) [reply core/profile.c:1664]: ok, understood.

**Comment by sfuchs79** [comment]
@dirkhh 
Could you please once recreate the translation source string? We have a couple of new strings and I would like to check if everything is ok after translations as well.

**Pull request merged**